app/src/test/java/org/simple/clinic/login/applock/AppLockScreenControllerTest.kt

```diff
@@ -88,4 +88,18 @@ class AppLockScreenControllerTest {
     uiEvents.onNext(AppLockScreenPinTextChanged("0"))
     verify(screen).setIncorrectPinErrorVisible(false)
   }
+
+  @Test
+  fun `when logout is clicked, the ui action should be triggered`() {
```

Comment:
Hmmm… `the ui action` sounds wrong

Reply:
Agreed. I just added it in here because I wanted to verify that the ui events are getting triggered. I plan to rename it when we actually implement the feature.